addons/membership/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    grade_id = fields.Many2one('res.partner.grade', 'Partner Level', tracking=True)
+    grade_label = fields.Char(compute='_compute_membership_type')
+    is_partnership_type = fields.Boolean(compute='_compute_membership_type')
+
+    def _compute_membership_type(self):
+        self.is_partnership_type = self.env['ir.config_parameter'].sudo().get_param('crm.membership_type') == 'partner'
+        if self.is_partnership_type:
+            self.grade_label = self.env._("Partner Level")
```

Comment:
When would we use the "partner level" naming (xcept at odoo because of our internal stuff) ? If you use membership, would say that we should always talk about Member Level, and simplify this whole code, isn't it ? It seems you are simplifying the code, so better stick on a single use case :)